bot/src/variables.ts

```diff
@@ -1,3 +1,4 @@
 export const ETHEREUM_NETWORK = process.env.ETHEREUM_NETWORK || 'kovan';
 export const KEEPER_MINIMUM_NET_PROFIT_DAI = parseInt(process.env.KEEPER_MINIMUM_NET_PROFIT_DAI || '');
 export const KEEPER_WALLET_PRIVATE_KEY = process.env.KEEPER_WALLET_PRIVATE_KEY;
+export const COLLATERAL_WHITELIST = process.env.WHITELISTED_COLLATERALS || false;
```

Comment:
Can you please rename it to `WHITELISTED_COLLATERALS` everywhere to stay consistent?